build.sbt

```diff
@@ -88,37 +88,63 @@ val publishJsonModuleSettings = List(
   publishLocal := publishLocal.dependsOn(publishM2).value
 )
 
-lazy val jsonConfig210 = project
+val testJSSettings = List(
+  testOptions += Tests.Argument(TestFrameworks.JUnit, "-v", "-a"),
+  scalaJSLinkerConfig ~= (_.withModuleKind(ModuleKind.CommonJSModule)),
+  libraryDependencies ++= List(
+    "com.github.plokhotnyuk.jsoniter-scala" %%% "jsoniter-scala-core" % Dependencies.jsoniterVersion,
+    "com.github.plokhotnyuk.jsoniter-scala" %%% "jsoniter-scala-macros" % Dependencies.jsoniterVersion % Provided
+  )
+)
+
+val testResourceSettings =
```

Comment:
Nit: can we enclose this whole expression with curly braces?

Reply:
Yep, fixed in 0a6fd08da1725e0b5aa3a39e51d5f42e00764152